base/src/main/java/com/smartdevicelink/proxy/rpc/SeatControlCapabilities.java

```diff
@@ -94,7 +94,7 @@ public String getModuleName() {
     /**
      * Sets the moduleName portion of the SeatControlCapabilities class
      *
-     * @param moduleName -  The short friendly name of the light control module. It should not be used to identify a module by mobile application.
+     * @param moduleName -  The short friendly name of the seat control module. It should not be used to identify a module by mobile application.
      */
     public SeatControlCapabilities setModuleName(@NonNull String moduleName) {
```

Comment:
Also, `AudioControlCapabilities` class seem to have similar typos in the Javadoc